CHANGELOG.md

```diff
@@ -8,57 +8,66 @@ This update focuses on new syntax features for MyST markdown, as well as a new c
 
 ### New
 
-**MyST Parser version 0.13.x**
+MyST Parser version 0.13.x
+
 : The MyST-NB and MyST-parser have both been upgraded.
   This comes with support for new syntax and a new configuration mechanism (see below for some examples).
 
-  See [the myst-parser changelog](https://myst-parser.readthedocs.io/en/latest/develop/_changelog.html#id4) for more information about the syntax additions.
+  (see also [the myst-parser changelog](https://myst-parser.readthedocs.io/en/v0.13.3/develop/_changelog.html#id4) for more information about the syntax additions)
+
+New `myst` extension configuration
 
-**New `myst` extension configuration**
 : The `myst_extended_syntax` configuration is **now deprecated**, in lieu of a more flexible extension mechanism.
   You may now enable individual `myst` extensions by adding them to the following section of your `_config.yml` file:
   ```yaml
   parse:
     myst_enable_extensions:
       - <list-of-extensions>
   ```
-  See [](content-blocks:myst-extensions) for more information.
+  See the [MyST syntax extension section here](docs/content/content-blocks.md) for more information.
+
+Citations and references configuration
 
-**Citations and references configuration**
 : This version comes with a version bump to `sphinxcontrib.bibtex v2.1.*`. This introduces new configuration for connecting your bibfiles (no longer using the bibliography directive), and makes the citation resolution process much more stable and dependable.
-  See [](content:citations) for more information, and the [`sphinxcontrib.bibtex` documentation](https://sphinxcontrib-bibtex.readthedocs.io/en/latest/) for more information about updates in the latest version.
+: See [](content:citations) for more information, and the [`sphinxcontrib.bibtex` documentation](https://sphinxcontrib-bibtex.readthedocs.io/en/latest/) for more information about updates in the latest version.
```

Comment:
```suggestion
: See [](content/citations.md) for more information, and the [`sphinxcontrib.bibtex` documentation](https://sphinxcontrib-bibtex.readthedocs.io/en/latest/) for more information about updates in the latest version.
```

Reply:
the only issue is that the actual link is half way down the page: https://deploy-preview-1199--jupyter-book.netlify.app/content/citations.html#citations-and-bibliographies.

For these ones I'm a bit undecided. As mentioned, you can get myst-parser to create header anchors: https://myst-parser.readthedocs.io/en/latest/using/syntax-optional.html#auto-generated-header-anchors, but we haven't used this yet in jupyter-book

Reply:
what do you think?

Reply:
ah I just saw this response sorry, IMO its ok to just link to the page, I think they can find it 🤷